pkg/operator/status.go

```diff
@@ -286,7 +286,7 @@ func (optr *Operator) syncUpgradeableStatus(co *configv1.ClusterOperator) error
 	if configNode.Spec.CgroupMode == configv1.CgroupModeV1 {
 		coStatusCondition.Status = configv1.ConditionFalse
 		coStatusCondition.Reason = "ClusterOnCgroupV1"
-		coStatusCondition.Message = "Cluster is using deprecated cgroup v1, which is removed in 4.19.  Please update the `CgroupMode` in the `nodes.config.openshift.io` object to 'v2'. This can be changed back to 'v1' while on 4.18, but must be 'v2' before you update to 4.19.  Once updated to 4.19, cgroup v1 is no longer an option"
+		coStatusCondition.Message = "Cluster is using deprecated cgroup v1, which is removed in 4.19.  Please update the `cgroupMode` in the `cluster` object of `nodes.config.openshift.io` resource type to 'v2'. This can be changed back to 'v1' while on 4.18, but must be 'v2' before you update to 4.19.  Once updated to 4.19, cgroup v1 is no longer an option. Please refer to the doc https://docs.redhat.com/en/documentation/openshift_container_platform/4.18/html/installation_configuration/enabling-cgroup-v1#nodes-clusters-cgroups-2-install_nodes-cluster-cgroups-1 to switch `cgroupMode` in OCP"
```

Comment:
minor nit: backticks for quoted code are familiar for devs from Markdown, but condition messages aren't Markdown.  I expect most customers will be more familiar with single (`'`) or double (`"`) quotes, and for some of these e.g. the nodes.config.openshift.io type), you can probably skip quoting entirely.  But I also think folks will understand what you're saying with your current backticks, so I'm ok with you leaving those in place if you don't find my reasoning convincing.

I think the docs link is a good idea, but you probably want to replace your current install-config link with [this post-install-config link][1].

[1]: https://docs.redhat.com/en/documentation/openshift_container_platform/4.18/html-single/nodes/index#nodes-clusters-cgroups-2_nodes-cluster-cgroups-2

Reply:
Addressed, Thanks

Reply:
44e4e4f52b8b covers all my nits; thanks!  I'll mark this thread resolved.